hkSymposium/source/ViewModel/MovieViewModel.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MovieViewModel.swift
+//  hkSymposium
+//
+//  Created by Anderson Santos Gusmão on 30/12/2017.
+//  Copyright © 2017 Anderson Santos Gusmão. All rights reserved.
+//
+
+import Foundation
+import TMDbClient
+
+class MovieViewModel: NSObject {
+
+    private var moviesList = [UpComingDataModel]()
+    private var pageIndex = 0
+
+    func getMovieTitle(for indexPath: IndexPath) -> String {
```

Comment:
Why do not you create a method to return a `movie` instead of all these methods?